.github/workflows/build-ci.yml

```diff
@@ -63,7 +67,7 @@ jobs:
                     restore-keys: ${{ matrix.os }}-composer-
             -   name: Install dependencies
                 run: |
-                    composer install --no-interaction
+                    composer update --no-interaction $([[ "${{ matrix.mode }}" == low-deps ]] && echo ' --prefer-lowest --prefer-stable')
```

Comment:
This is a bit different than what we do in PHPLIB IIRC, where the CI parameter is just the args to inject (and defaults to an empty string).

I'm sure this works though so no objection. The approach in PHPLIB also doesn't change the build title, which you've achieved here.